rca/utils/templatetags/util_tags.py

```diff
@@ -106,6 +105,9 @@ def social_media_links(context):
     "beta.rca.ac.uk",
 ]
 
+if hasattr(settings, "WAGTAILADMIN_BASE_URL"):
+    default_domains.append(settings.WAGTAILADMIN_BASE_URL)
+
```

Comment:
At build time WAGTAILADMIN_BASE_URL isn't set but at run time it should be. There's an env var of `PRIMARY_HOST` available.